File: py/equation_grapher.py
from sympy import symbols
from sympy.plotting import plot

x = symbols('x')
y = symbols('y')

print("Equation Grapher v1.0")
print("Copyright (c) 2020 miniprime1")
print("[Variables: (x or y)]\n")

eq = input('Enter Equation: ')
print()

try: 
    plot(eq)
except Exception as e: 
    print("Error!\n" + str(e))

File: matlab/Document/EquationGrapher.m
disp("Equation Grapher v1.0")
disp("Copyright (c) 2020 miniprime1")
disp("[Variables: (x or y)]")
disp(" ")

syms x y
eq = input('Enter Equation: ');
disp(" ")

try
    ezplot(eq)
    disp("Successful!")
catch
    disp("Error!")
end

clear